src/core/ix86/iR3000A.cc

```diff
@@ -661,35 +657,28 @@ bool X86DynaRecCPU::Init() {
     memcpy(m_psxRecLUT + 0x9fc0, &m_psxRecLUT[0x1fc0], 8 * sizeof(uintptr_t));
     memcpy(m_psxRecLUT + 0xbfc0, &m_psxRecLUT[0x1fc0], 8 * sizeof(uintptr_t));
 
-    gen.x86Init(m_recMem);
-
     return true;
 }
 
 void X86DynaRecCPU::Reset() {
     R3000Acpu::Reset();
 
-    memset(m_recRAM, 0, 0x200000);
-    memset(m_recROM, 0, 0x080000);
-
-    gen.x86Init(m_recMem);
+    std::memset(m_recRAM, 0, 0x200000);
```

Comment:
Careful about the ability to have 8MB working properly: https://github.com/grumpycoders/pcsx-redux/blob/main/src/core/psxemulator.h#L176

Reply:
Should be good now.